Foxtrot/Tests/CRASanitizeTest.cs

```diff
@@ -132,81 +122,107 @@ private void CheckSourceFiles(string v9proj, string v10proj)
       var onlyInV9 = v9Sources.Where(source => !v10Sources.Contains(source));
       var onlyInV10 = v10Sources.Where(source => !v9Sources.Contains(source));
 
-      Console.WriteLine("Checking project sources {0} vs. {1}", v9proj, v10proj);
+      _testOutputHelper.WriteLine("Checking project sources {0} vs. {1}", v9proj, v10proj);
       var v9errors = onlyInV9.Count();
       foreach (var source in onlyInV9)
       {
-        Console.WriteLine("File {0} only in {1}", source, v9proj);
+        _testOutputHelper.WriteLine("File {0} only in {1}", source, v9proj);
       }
 
       var v10errors = onlyInV10.Count();
       foreach (var source in onlyInV10)
       {
-        Console.WriteLine("File {0} only in {1}", source, v10proj);
+        _testOutputHelper.WriteLine("File {0} only in {1}", source, v10proj);
       }
 
-      Assert.AreEqual(0, v9errors, "Found {0} errors in project file {1}", v9errors, v9proj);
-      Assert.AreEqual(0, v10errors, "Found {0} errors in project file {1}", v10errors, v10proj);
+      Assert.Equal(0, v9errors);
```

Comment:
The same true here.

Without diagnostic the only way to find this stuff is to look at stack trace. Proper messages in the asserts always helps to find the root cause faster.